### PR TITLE
FSAR-30: Back to top.

### PR DESCRIPTION
**Pull request by beloglazov91**
Please provide a thorough description of your component and its usage with screenshots.

## Checklist before opening a PR

Before opening a PR, please make sure:

- [ ] your work is in a branch based on and synced with the develop branch.
- [ ] your scss and twig files are imported in the javascript file. Without this, Webpack won't be able to compile them.
- [ ] your newComponent is imported in the index.js file.
- [ ] your folder hierarchy follows the proper structure.
- [ ] your code follows the coding standards and naming conventions defined.
- [ ] your code has been tested as AA accessibility compliant to the WCAG 2.1 AA standard.
- [ ] your code has been tested in all the browsers listed in the GDS browser's list, including IE11.
- [ ] your code has been tested in all the different screensizes defined.

### CODE REVIEW

File: src/components/general/BackToTop/backToTop.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import './backToTop.html.twig';
+import './backToTop.scss';
+import smoothscroll from 'smoothscroll-polyfill';
+
+export default function () {
+    smoothscroll.polyfill();
+
+    window.addEventListener('DOMContentLoaded', function () {
+        const backToTop = document.querySelector('.back-to-top__link');
+        console.log(backToTop);
```
Comment (c-ancia) [inline src/components/general/BackToTop/backToTop.js:10]: can you remove the console.log please? 🙂

**Review by c-ancia — CHANGES_REQUESTED** [review]

File: src/components/general/BackToTop/backToTop.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  &__link {
+    @include link;
+    display: table;
+
+    &::before {
+      content: "";
+      background-image: url("./assets/arrow-up.svg");
+      display: inline-block;
+      width: 1rem;
+      height: 1.55rem;
```
Comment (c-ancia) [inline src/components/general/BackToTop/backToTop.scss:13]: assuming it's the fixed height of the file?

Comment (beloglazov91) [reply src/components/general/BackToTop/backToTop.scss:13]: Yes.

File: src/components/general/BackToTop/backToTop.js (hunk 1 above)
Comment (beloglazov91) [reply src/components/general/BackToTop/backToTop.js]: Done :)

**Pull request merged**